Gruntfile.js

```diff
@@ -14,7 +14,9 @@ const extract = require('jsxgettext-recursive');
 // use this if you want to recursively match all subfolders:
 // 'test/spec/**/*.js'
 module.exports = function (grunt) {
-  'use strict';
+  'use strict'
```

Comment:
`'semi': [2, 'never']` (default for "fxa/server" config) will fail fast and hard if you dare bring a semicolon to a knife fight.